cfg_mgmt/util.py

```diff
@@ -18,6 +19,9 @@
 
 logger = logging.getLogger(__name__)
 
+MAX_RETRIES = 5
```

Comment:
I think we need more retries / longer delay (or (better)  exponential backoff).

consider we sometimes see maintenance windows of a couple of hours for GHE. so we should make our code be able to bridge such a gap

also, instead of setting module-level-constants, and read from multiple places in (one) function, preferably set those as default values in function-parameters.